include/zephyr/devicetree.h

```diff
@@ -3319,6 +3425,17 @@
 #define DT_INST_STRING_UPPER_TOKEN(inst, prop) \
 	DT_STRING_UPPER_TOKEN(DT_DRV_INST(inst), prop)
 
+/**
+ * @brief Get a `DT_DRV_COMPAT` instance's string property's value as
```

Comment:
Same here

Reply:
property string name -> property name, fixed also in other macros.